docs/pages/repo/docs/handbook/deploying-with-docker.mdx

```diff
@@ -152,6 +152,7 @@ COPY --from=installer /app/apps/web/package.json .
 # https://nextjs.org/docs/advanced-features/output-file-tracing
 COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/standalone ./
 COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/static ./apps/web/.next/static
+COPY --from=installer --chown=nextjs:nodejs /app/apps/web/public ./apps/web/public
```

Comment:
Can you update this in the `with-docker` example as well?

Reply:
Done

Reply:
I  don't think `standalone` mode works with the latest Next.js 13

Reply:
Actually it works with Next.js 13.
If you're using `turborepo` and docker then you need to add following in your `next.config.js`:
```js
const path = require("path");
const withTM = require("next-transpile-modules")(["ui"]);

module.exports = withTM({
  reactStrictMode: true,
  output: "standalone",
  experimental: {
    outputFileTracingRoot: path.join(__dirname, "../../"),
  },
});
```

`outputFileTracingRoot` is important for this `Dockerfile` to work correctly. It would be great if this was speciefied in `turborepo` docs.